api/envoy/admin/v2alpha/config_dump.proto

```diff
@@ -205,60 +257,76 @@ message ScopedRoutesConfigDump {
     string version_info = 2;
 
     // The scoped route configurations.
-    repeated api.v2.ScopedRouteConfiguration scoped_route_configs = 3;
+    repeated api.v2.ScopedRouteConfiguration scoped_routes = 3;
```

Comment:
Prefer to avoid gratuitous renames, even within an alpha version, but that's just me :P

Reply:
Does it count as a rename given it's now in a new encapsulating message?
I figure as long as we're moving things around better to have consistency but I'm Ok with changing back too

Reply:
Per above my preference would be to rename here to the best name since we are doing substantial movement anyway.